int32 transceiverID
int32 responseID
string transponderModelName
string data